### ISSUE TITLE
Support css color level 4 function

### ISSUE THREAD

**Issue by MoOx** [harthur/color#55]
http://dev.w3.org/csswg/css-color/#modifying-colors

See https://github.com/ianstormtaylor/css-color-function

https://github.com/ianstormtaylor/css-color-function/blob/master/lib/adjusters.js

**Comment by Qix-** [comment]
This might belong in `color-string`. I'll look a bit further into it later.

**Comment by kevinSuttle** [comment]
Does this cover everything? https://github.com/postcss/postcss-color-function

**Comment by Qix-** [comment]
Yeah - that's a great reference. Not going to make it into the v1.x makeover but I'll open a ticket over there :)

**Comment by Qix-** [comment]
It was ultimately removed and probably a bit out of scope for this package anyway. After 6 years, going to close this.
